src/zks_wallet/wallet.rs

```diff
@@ -28,8 +30,31 @@ use ethers::{
     },
 };
 use ethers_contract::providers::PendingTransaction;
-use serde_json::Value;
+use lazy_static::lazy_static;
+use serde_json::{Map, Value};
+use std::collections::HashMap;
 use std::{fs::File, io::BufReader, path::PathBuf, str::FromStr, sync::Arc};
+use zksync_web3_rs::core::abi::Tokenize;
+
+const RAW_ERC20_DEPOSIT_GAS_LIMIT: &str = include_str!("DepositERC20GasLimit.json");
+
+lazy_static! {
```

Comment:
We should consider using `once_cell` instead of `lazy_static` since the first one is widely recommended over the lazy macro and it's progressively being added to the `std`

Reply:
I think that we can create a ticket to refactor this in the future.

Reply:
Yep, sounds reasonable

Reply:
Created issue @ https://github.com/lambdaclass/zksync-web3-rs/issues/102